Makefile

```diff
@@ -58,12 +58,13 @@ distclean: clean
 
 # Lint JavaScript source files.
 lint: install
-	@$(ESLINT) $(ALL_FILES)
```

Comment:
If we're not using ESLint anymore in the makefile, we could remove it from line 5 as well.

Reply:
Good catch, fixed

Reply:
Today's free tip: with newer versions of yarn you can directly invoke `bin` programs.
Example: `yarn codecov` instead of `./node_modules/.bin/codecov`